code/modules/xenoarcheaology/finds/find_spawning.dm

```diff
@@ -520,6 +522,62 @@
 			if(prob(25))
 				apply_material_decorations = FALSE
 			new_item = new /obj/item/weapon/telecube/randomized(src.loc)
+			item_type = new_item.name
+
+		if(40)
```

Comment:
These should really be using the #defines that you set above, rather than the magic numbers they evaluate to.